src/Bicep.LangServer/Completions/IPublicRegistryModuleMetadataProvider.cs

```diff
@@ -11,10 +11,13 @@
 
 namespace Bicep.LanguageServer.Providers
 {
+    public record PublicRegistryModule(string name, string? description, string? documentationUri);
+    public record PublicRegistryModuleVersion(string version, string? description, string? documentationUri);
```

Comment:
formatting: Should use PascalCase for positional parameters.

Reply:
d8c1e0098fb0b23f3acd942b088ed458fcb8c1ce